src/components/Item.js

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+
+export default function Item(props){
+    const [showSignUpBtn, setShowSignUpBtn] = useState(true);
+    const [supplier, setSupplier] = useState("");
+
+    let item = props.props.item;
+    let user = props.props.user;
+    let hostId = props.props.host;
+
+    async function signUp(){
+        console.log(JSON.stringify(item));
+        await fetch(`http://localhost:8080/items/${item.id}`,{
+            body:JSON.stringify(item),
+            method:"PUT",
+            headers:{
+                "Content-Type":"application/json"
+            }
+        });
+    }
+
+    async function removeItem(){
+        await fetch(`http://localhost:8080/items/${item.id}`,{
+            body:JSON.stringify(item),
+            method:"DELETE",
+            headers:{
+                "Content-Type":"application/json"
+            }     
+        });
+    }
+
+    function updateSupplier(event){
+        setSupplier(event.target.value);
+    }
+
+    function signUpInput(){
```

Comment:
Changed to just show the same thing whether user or guest. So a user can type in a different name if they want